nfts/scripts/lockUnlockToken.js

```diff
@@ -3,8 +3,7 @@
 import fetch from "node-fetch";
 import env from "dotenv";
 import { exit } from "process";
-import getDeveloperWallet, {
-  sleep,
+import {
```

Comment:
the default image doesn't look good. Let's update it with the image that you have in the readme docs 
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/100057389/193884607-0e8eab59-0f44-49f3-9839-13c8f0503cac.png">

Reply:
fixed